includes/controllers/class.llms.controller.lesson.progression.php

```diff
@@ -162,8 +171,17 @@ public function handle_incomplete_form() {
 			return;
 		}
 
+		/**
+		 * Filter to modify the User id instead of current loggedin user id.
+		 *
+		 * @param int $user_id User id to mark lesson as incomplete.
+		 *
+		 * @since 5.3.4
+		 */
+		$user_id = apply_filters( 'llms_lesson_in_complete_user_id', get_current_user_id() );
```

Comment:
It's "incomplete" not "in complete" so separation via underscore is incorrect.

```suggestion
		$user_id = apply_filters( 'llms_lesson_incomplete_user_id', get_current_user_id() );
```

Reply:
Done 👍